src/main/java/com/esri/samples/portal/oauth/OAuthChallengeHandler.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.esri.samples.portal.oauth;
+
+import com.esri.arcgisruntime.security.*;

```

Comment:
We don't use * imports in the main code base but maybe it is allowed in samples.